daemon/execdriver/windows/exec.go

```diff
@@ -13,13 +13,14 @@ import (
 	"github.com/natefinch/npipe"
 )
 
-func (d *driver) Exec(c *execdriver.Command, processConfig *execdriver.ProcessConfig, pipes *execdriver.Pipes, startCallback execdriver.StartCallback) (int, error) {
+// Exec implements the exec driver Driver interface.
```

Comment:
Since all these methods in different exec drivers have almost same purpose, and their docs are mostly the same, so as suggested in https://github.com/docker/docker/pull/14848#issuecomment-124560185, #14848 expended docs in Driver interface, docs for method like `Exec` can be found in the interface, we can only maintain pertinent details required for these particular implementations.